src/main/java/xyz/vegaone/easytrackingv2/domain/UserStoryEntity.java

```diff
@@ -33,6 +34,9 @@
     @JoinColumn(name = "user_id")
     private UserEntity user;
 
+    @OneToMany(mappedBy = "userStory")
+    private List<TaskEntity> taskEntityList;
```

Comment:
To work out of the box, the mapper needs the dto fields and the entity fields to match. E.g. TaskEntity field name in UserStoryEntity should match Task in UserStory.